src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerOptions.cs

```diff
@@ -11,34 +11,47 @@ namespace Microsoft.Extensions.Logging.Console
     /// </summary>
     public class ConsoleLoggerOptions
     {
-        private ConsoleLoggerFormat _format = ConsoleLoggerFormat.Default;
-
-        /// <summary>
-        /// Includes scopes when <see langword="true" />.
-        /// </summary>
-        public bool IncludeScopes { get; set; }
-
         /// <summary>
         /// Disables colors when <see langword="true" />.
         /// </summary>
+        [System.ObsoleteAttribute("ConsoleLoggerOptions.DisableColors has been deprecated. Please use ColoredConsoleFormatterOptions.DisableColors instead.", false)]
         public bool DisableColors { get; set; }
 
         /// <summary>
         /// Gets or sets log message format. Defaults to <see cref="ConsoleLoggerFormat.Default" />.
         /// </summary>
+        [System.ObsoleteAttribute("ConsoleLoggerOptions.Format has been deprecated. Please use ConsoleLoggerOptions.FormatterName instead.", false)]
         public ConsoleLoggerFormat Format
         {
-            get => _format;
+#pragma warning disable CS0618
+            get
+            {
+                if (FormatterName != null && FormatterName.Equals(ConsoleFormatterNames.Systemd, StringComparison.OrdinalIgnoreCase))
+                    return ConsoleLoggerFormat.Systemd;
+                return ConsoleLoggerFormat.Default;
+            }
             set
             {
-                if (value < ConsoleLoggerFormat.Default || value > ConsoleLoggerFormat.Systemd)
+                if (value == ConsoleLoggerFormat.Systemd)
```

Comment:
>== [](start = 26, length = 2)

should we compare this as ordinal ignore casing?

Reply:
this is an enum though

Reply:
oh, I can see how will be confusing the old and new interfaces :-) thanks.